scripts/preinstall-script.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+if (process.env.npm_execpath.indexOf('yarn') === -1) {
+  console.error(
+    '\x1b[41m',
+    'Error: You must use yarn to install dependencies',
+    '\x1b[0m',
+    '\n'
+  )
+  process.exit(1)
+}
+
+console.warn(
+  '\x1b[36m%s\x1b[0m',
+  '\nFor this project we recommend using yarn install package-name instead of npm install\n'
```

Comment:
@victorsingh just realized the message might be misleading since yarn does not have 'install', I would suggest instead we point them to the CONTRIBUTING.md#setup section so they can read more here on why we are using yarn. 

The link would be: https://github.com/ViacomInc/data-point/blob/master/CONTRIBUTING.md#setup


does this make sense?

Reply:
Yea that makes more sense. It would also encourage more people to read CONTRIBUTING.md